game-core/build.gradle

```diff
@@ -115,10 +115,12 @@ dependencies {
     compileOnly group: 'org.projectlombok', name: 'lombok', version:'1.16.20'
     runtime remoteFile('https://github.com/kirill-grouchnikov/substance/raw/master/drop/7.1.01/trident-1.4.jar')
 
+    testImplementation 'org.junit.jupiter:junit-jupiter-api:5.1.0'
+    testRuntimeOnly 'org.junit.jupiter:junit-jupiter-engine:5.1.0'
```

Comment:
If we're updating the version here, we should update it everywhere else in the script.

Reply:
good call, didn't realized there was a third; fixed in: 8376548